File: C# Nutriologia/Clases/ConexionBD.cs
using Microsoft.VisualBasic;
using System;
using System.Data.SqlClient;

namespace C__Nutriologia
{
    public class ConexionBD
    {
        public static SqlConnection conn = new SqlConnection();

        public static void conexion()
        {
            try
            {
                conn.ConnectionString = @"Data Source = ASREN-PC\SQLSERVER; Initial Catalog = BD_Nutriologo; 
            Integrated Security = True";

                conn.Open();
            }
            catch (Exception ex)
            {
                Interaction.MsgBox("No se conecto a la base de datos...", Constants.vbInformation, ex.Message);
            }
        }
}

File: Nutriologia/Modulos/ConexionBD.vb
Imports System.Data.SqlClient
Module ConexionBD
    Public conn As New SqlConnection

    Public Sub conexion()
        Try
            conn.ConnectionString = "Data Source = ASREN-PC\SQLSERVER; Initial Catalog = BD_Nutriologo; 
            Integrated Security = True"

            conn.Open()
        Catch ex As Exception
            MsgBox("No se conecto a la base de datos...", vbInformation, ex.Message)
        End Try
    End Sub
End Module